extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -401,6 +402,10 @@ describes.realWin('amp-story', {
     it('should pause the story if there is a consent', () => {
       sandbox.stub(win.history, 'replaceState');
 
+      sandbox.stub(Services, 'actionServiceForDoc')
+          .returns({setWhitelist: () => {}, trigger: () => {}});
+      sandbox.stub(AmpStoryConsent.prototype, 'buildCallback');
```

Comment:
Can you add a comment to each of these stubs explaining why you're doing this?

Reply:
Done